.gitignore

```diff
@@ -34,3 +34,8 @@ Pipfile.lock
 .coverage
 .python-version
 .coverage.*
+.DS_Store
+
+data/example_dataset/raw/
```

Comment:
This should be changed to whatever it's called in toffy

Reply:
Looks like it's just called `combined` in [toffy](https://github.com/angelolab/toffy/tree/main/toffy/data). So it should be `data/example_dataset/combined`?

Reply:
Found it, it's `image_data`
<img width="606" alt="image" src="https://user-images.githubusercontent.com/13770365/189005441-62df7b17-fca6-4317-bf62-528baa4207b2.png">